File: core/src/main/java/com/graphhopper/routing/ev/MaxSlope.java
package com.graphhopper.routing.ev;

/**
 * Maximum elevation change in m/100m.
 */
public class MaxSlope {
    public static final String KEY = "max_slope";

    public static DecimalEncodedValue create() {
        return new DecimalEncodedValueImpl(KEY, 5, 0, 1, true, false, false, true);
    }
}

File: core/src/main/kotlin/com/graphhopper/routing/ev/MaxSlope.kt
package com.graphhopper.routing.ev

/**
 * Maximum elevation change in m/100m.
 */
object MaxSlope {
    const val KEY = "max_slope"

    @JvmStatic
    fun create(): DecimalEncodedValue = DecimalEncodedValueImpl(KEY, 5, 0.0, 1.0, true, false, false)
}
